By default enforce check of clone image via allow-list by image name and tag (#4056)
woodpecker-ci/woodpecker: docs/docs/30-administration/10-server-config.md

> Defaults are defined in [shared/constant/constant.go](https://github.com/woodpecker-ci/woodpecker/blob/main/shared/constant/constant.go)

Docker images to run in privileged mode. Only change if you are sure what you do!

> You should specify the tag as images are compared via exact matches.

Author: ```suggestion
You should specify the tag of your images too, as this enforces exact matches.
```
